### PR TITLE
Food fixes [Ready]

### PR DESCRIPTION
**Pull request by Petethegoat**
Code cleanliness and some minor fixes for new food stuff.
Adds 500 colouring for more human onmobs.
Added the ability to save things from the fryer.
Fried items are put into the holder object as well now, which should allow admins to retrieve them if they're not eaten.

### CODE REVIEW

File: code/game/machinery/kitchen/deep_fryer.dm [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
-	var/on = 0 // Is it deep frying already?
+	var/on = FALSE	//Is it deep frying already?
 
-/obj/machinery/deepfryer/attackby(var/obj/item/O as obj, var/mob/user as mob)
+/obj/machinery/deepfryer/attackby(obj/item/I, mob/user)
 	if(on)
-		user << "The machine is already active, please wait."
+		user << "<span class='notice'>[src] is already active, please wait.</span>"
+		return
+	if(istype(I, /obj/item/weapon/reagent_containers/food/snacks/deepfryholder))
```
Comment (Iamgoofball) [inline code/game/machinery/kitchen/deep_fryer.dm:23]: recursive deep frying is intentional

**Comment by Iamgoofball** [comment]
other then that :+1:

File: code/game/machinery/kitchen/deep_fryer.dm (hunk 1 above)
Comment (Petethegoat) [inline code/game/machinery/kitchen/deep_fryer.dm:23]: but it adds nutriment each time. i'm pretty sure we'd run into issues with name length and stupid exploits too.

i think being able to grille, deepfry, soupify and then cerealify stuff is more than enough

Comment (Iamgoofball) [inline code/game/machinery/kitchen/deep_fryer.dm:23]: Well, alright then.

**Comment by Petethegoat** [comment]
Ready for now.

**Comment by Atskadan** [comment]
Nice.

**Comment by Iamgoofball** [comment]
code/game/machinery/kitchen/deep_fryer.dm:17:error: user: undefined var
@Petethegoat

**Comment by Petethegoat** [comment]
durrr

**Pull request merged**